### ISSUE TITLE
Utilizing show_generator_output and obtaining loss values

### ISSUE THREAD

**Issue by sahiljuneja** [Brok-Bucholtz/Project-Face-Generation#5]
It might help if it's mentioned how to utilize the function. 

Many students might end up adding it directly and try to run the function for every batch. Which might slow down their training perhaps? So clarifying, display it after N iterations might do them good. And this might help reviewers as well. Lots of images increase file size and can be slow to upload depending on their internet connection :)

Also, providing them with code that calculates and prints out loss might be good. 

It seems you want the students to try to implement this on their own this time - all the training code. Which seems a bit off considering the previous projects didn't have that. If that is the case, then perhaps a bit more detail on what they can do to help themselves more to get the most out of training (in terms of identifying the problems and what specifically to add in that case) would be good for them.  Like printing out loss values as training progresses (not just after each epoch for example) or plotting the loss values [an issue I reported for last week's lessons on GANs in this regard] and being able to relate that to how well the model performs etc.

**Comment by sahiljuneja** [comment]
Regarding the show_generator_output function. 

Everytime it will output different images during training, right?

Isn't it better for the students to be able to visualize images such that it shows how they improve throughout the training till it produces the faces or digits? 

Correct me if I am interpreting the above incorrectly. But my training progress seems to indicate the above.

**Comment by Brok-Bucholtz** [comment]
We added a warning about over using the `show_generator_output` function in commit https://github.com/Brok-Bucholtz/P5-Beta/commit/61f02d4206e8a568c709a5923e925f22b83ed9df.

We also added added information on what the loss should be in commit https://github.com/Brok-Bucholtz/P5-Beta/commit/a1783eaf1583fc968c3a5d40aa6655db89f15bb0.  This should give them a hint that they need to print the loss.

@sahiljuneja,

We use a different z input each time to get a better idea of how well the GANs is actually performing.  Are you proposing using the same z value for the period of training or also between train sessions?

Thanks for your input!

**Comment by sahiljuneja** [comment]
I was proposing the same value such that students can sequentially see how a particular face is getting generated [does that make sense?] 

I will try to run it again today and try to get better results to give an example image set here on what I mean.
